sites/all/modules/ereol_article/ereol_article.views_default.inc

```diff
@@ -84,23 +85,51 @@ function ereol_article_views_default_views() {
   $handler->display->display_options['exposed_form']['options']['exposed_sorts_label'] = 'Sortér efter';
   $handler->display->display_options['exposed_form']['options']['sort_asc_label'] = 'Stigende';
   $handler->display->display_options['exposed_form']['options']['sort_desc_label'] = 'Faldende';
-  $handler->display->display_options['exposed_form']['options']['autosubmit'] = TRUE;
   $handler->display->display_options['exposed_form']['options']['bef'] = array(
     'general' => array(
+      'input_required' => 0,
+      'text_input_required' => array(
+        'text_input_required' => array(
+          'value' => 'Vælg et filter og klik på Anvend for at se resultater',
```

Comment:
Should this be English?

Reply:
I have no idea why `drush features-update` generates texts in Danish. I could manually change it, but I risk breaking more than I'm fixing.